src/main/java/codeu/model/data/User.java

```diff
@@ -23,39 +23,47 @@
   private final String name;
   private final String password;
   private final Instant creation;
-
+  private final String About_Me_Text; //String for About Me Section
```

Comment:
Would be great to name the variable in the same fashion as the others. For eg. aboutMeText in this case.